project/MimaSettings.scala

```diff
@@ -0,0 +1,22 @@
+import com.typesafe.tools.mima.plugin.MimaKeys._
+import com.typesafe.tools.mima.plugin.MimaPlugin
+import sbt.Keys._
+import sbt._
+
+object MimaSettings {
+
+  // If new version introduces breaking changes,
+  // clear-out mimaBinaryIssueFilters and mimaPreviousVersions.
+  // Otherwise, add previous version to set without
+  // removing other versions.
+  val mimaPreviousVersions = Set()
```

Comment:
So, we'll have to add it manually after every release? Is there a way to automate it? Otherwise we'll need to have a standardized place for "Release checklist"? Wiki? Repo itself? (IMO its related to DQA, @dilyand)

Reply:
I thought and looked for a way to automate but there is no clear way for this as far as I understand. One of the reason is that these previous versions should be determined by the breaking changes in the current version. So, let's say we bumped from 1.0.0 to 1.0.1 and there is no breaking changes obviously, in that case 1.0.0 should be added to `mimaPreviousVersions`. However, in case of bumping from 1.0.1 to 2.0.0 which most probably there will be breaking changes, `mimaPreviousVersions` should be cleared-out. So, maybe some scripts can be written for checking current version and place previous versions according to that but I could not be sure about it.

Reply:
Yeah, I think we just should have a "Release checklist" wikii page somehwere, because people will keep forgetting about it.